### PR TITLE
Increase time for shutdown and remove skips

### PR DESCRIPTION
**Pull request by fedorovpv**
[Lenovo](https://ci-dev.vedenemo.dev/job/ghaf-hw-test-manual/6301/artifact/Robot-Framework/test-suites/SP-T75-4ORSP-T75-5/log.html)

### CODE REVIEW

File: Robot-Framework/test-suites/gui-tests/gui_power.robot [hunk 1]
```diff
@@ -51,54 +51,38 @@ Reboot from power menu
...
     ...               Check that it shuts down and then wakes up with a short power button press.
     [Tags]            SP-T75  SP-T75-5  lenovo-x1  darter-pro  lab-only
     Select power menu option   x=925   y=120   confirmation=True
+    Set Test Variable          ${max_elapsed}  25
+
     ${start_time}     Get Time    epoch
     ${end_time}       Wait Until Device Is Down
     ${elapsed}        Evaluate    ${end_time} - ${start_time}
     Log               Shutdown took ${elapsed} seconds   console=True
     # After shutdown always wait at least until 30 seconds and for 10 more seconds if shutdown was faster than 20 seconds
```
Comment (milva-unikie) [inline Robot-Framework/test-suites/gui-tests/gui_power.robot]: This comment should be updated or maybe just removed

File: Robot-Framework/test-suites/gui-tests/gui_power.robot [hunk 2]
```diff
@@ -51,54 +51,38 @@ Reboot from power menu
...
-    IF    ${elapsed} <= 20
+    IF    ${elapsed} <= ${max_elapsed}
         ${wait_time}  Evaluate    30 - ${elapsed}
```
Comment (milva-unikie) [inline Robot-Framework/test-suites/gui-tests/gui_power.robot]: To make sure that the device is actually down before it is booted again, increasing 30 to 35 would be a good change too. Or that could be calculated from the `${max_elapsed}`

File: list_of_skipped_tests.md
Comment (milva-unikie) [inline list_of_skipped_tests.md]: This line can be removed too

File: Robot-Framework/test-suites/gui-tests/gui_power.robot (hunk 2 above)
Comment (leivos-unikie) [reply Robot-Framework/test-suites/gui-tests/gui_power.robot]: Could it be 
`${wait_time}    Evaluate    ${max_elapsed} - ${elapsed} + 5`

Just thinking the edge case it appears to be down just before ${max_elapsed}

**Review by milva-unikie — APPROVED** [review]

File: Robot-Framework/test-suites/gui-tests/gui_power.robot (hunk 2 above)
Comment (fedorovpv) [reply Robot-Framework/test-suites/gui-tests/gui_power.robot]: when max was 20 it was  `30 - ${elapsed},` so I think now it should be more like` {max_elapsed} - ${elapsed} + 10`
